### PR TITLE
Feat btn user location

### PR DESCRIPTION
**Pull request by TeeWhyKay**
## Why

This pull request is needed because:

' Functionality to allow user to supply their location '

[Link to Card]()

## What

This pull request introduces the following:

 * `Removed the search button, added the location button to enable user to specify location`
 *  Search is done when pressing enter

`Add Implementation details if needed`

### Screenshot

image

## How to Test

`1. Press on the location icon in the search field
`2. A popup will ask for your location, press allow

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -27,10 +27,12 @@ import "bootstrap";
 
 // Internal imports, e.g:
 import { initModal } from '../plugins/init_modal';
+import { initLocation } from '../plugins/init_location';
 
 document.addEventListener('turbolinks:load', () => {
   // Call your functions here, e.g:
   initModal();
+  initLocation(); // only work in the home page
```
Comment (mxng) [inline app/javascript/packs/application.js]: Do you mean like only needs to work in the home page?

Comment (mxng) [reply app/javascript/packs/application.js]: Remove comment pls

**Review by mxng — COMMENTED** [review]
Needs minor update

File: app/javascript/packs/application.js (hunk 1 above)
Comment (TeeWhyKay) [reply app/javascript/packs/application.js]: removed

**Review by TeeWhyKay — COMMENTED** [review]
removed comment

**Review by mxng — APPROVED** [review]
Looks ok!

**Pull request merged**